src/wfs/src/test/java/org/geoserver/wfs/GetFeatureTest.java

```diff
@@ -398,6 +398,24 @@ public void testStrictComplianceBBoxValidator() throws Exception {
     }
 
     @Test
+    public void testStrictComplianceVersionNegotiation() throws Exception {
```

Comment:
Yep, I started with a GetFeature test, then noticed it was supposed to be getCapabilities and forgot to move it. Thanks for the reminder.